2022-09-27 14:48:16,844 INFO    StreamThr :85950 [internal.py:wandb_internal():93] W&B internal server running at pid: 85950, started at: 2022-09-27 14:48:16.841440
2022-09-27 14:48:16,847 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: status
2022-09-27 14:48:16,850 DEBUG   SenderThread:85950 [sender.py:send_request():316] send_request: status
2022-09-27 14:48:16,853 INFO    WriterThread:85950 [datastore.py:open_for_write():75] open: /home/xush/Project/AICompetition/feedback-prize/code/wandb/run-20220927_144816-3979rmin/run-3979rmin.wandb
2022-09-27 14:48:16,854 DEBUG   SenderThread:85950 [sender.py:send():302] send: header
2022-09-27 14:48:16,854 DEBUG   SenderThread:85950 [sender.py:send():302] send: run
2022-09-27 14:48:17,696 INFO    SenderThread:85950 [dir_watcher.py:__init__():216] watching files in: /home/xush/Project/AICompetition/feedback-prize/code/wandb/run-20220927_144816-3979rmin/files
2022-09-27 14:48:17,696 INFO    SenderThread:85950 [sender.py:_start_run_threads():916] run started: 3979rmin with start time 1664261296.834918
2022-09-27 14:48:17,700 DEBUG   SenderThread:85950 [sender.py:send():302] send: summary
2022-09-27 14:48:17,700 INFO    SenderThread:85950 [sender.py:_save_file():1156] saving file wandb-summary.json with policy end
2022-09-27 14:48:17,701 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: check_version
2022-09-27 14:48:17,701 DEBUG   SenderThread:85950 [sender.py:send_request():316] send_request: check_version
2022-09-27 14:48:18,064 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: run_start
2022-09-27 14:48:18,077 DEBUG   HandlerThread:85950 [meta.py:__init__():35] meta init
2022-09-27 14:48:18,077 DEBUG   HandlerThread:85950 [meta.py:__init__():50] meta init done
2022-09-27 14:48:18,077 DEBUG   HandlerThread:85950 [meta.py:probe():216] probe
2022-09-27 14:48:18,084 DEBUG   HandlerThread:85950 [git.py:repo():40] git repository is invalid
2022-09-27 14:48:18,084 DEBUG   HandlerThread:85950 [meta.py:_save_pip():54] save pip
2022-09-27 14:48:18,085 DEBUG   HandlerThread:85950 [meta.py:_save_pip():68] save pip done
2022-09-27 14:48:18,085 DEBUG   HandlerThread:85950 [meta.py:_save_conda():75] save conda
2022-09-27 14:48:18,091 ERROR   HandlerThread:85950 [meta.py:_save_conda():84] Error saving conda packages
Traceback (most recent call last):
  File "/home/xush/miniconda3/envs/EnglishLearnTorch/lib/python3.6/site-packages/wandb/sdk/internal/meta.py", line 81, in _save_conda
    ["conda", "env", "export"], stdout=f, stderr=subprocess.DEVNULL
  File "/home/xush/miniconda3/envs/EnglishLearnTorch/lib/python3.6/subprocess.py", line 287, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/home/xush/miniconda3/envs/EnglishLearnTorch/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/home/xush/miniconda3/envs/EnglishLearnTorch/lib/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'conda': 'conda'
2022-09-27 14:48:18,093 DEBUG   HandlerThread:85950 [meta.py:_save_conda():85] save conda done
2022-09-27 14:48:18,093 DEBUG   HandlerThread:85950 [meta.py:probe():254] probe done
2022-09-27 14:48:18,098 DEBUG   SenderThread:85950 [sender.py:send():302] send: files
2022-09-27 14:48:18,099 INFO    SenderThread:85950 [sender.py:_save_file():1156] saving file wandb-metadata.json with policy now
2022-09-27 14:48:18,110 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: stop_status
2022-09-27 14:48:18,111 DEBUG   SenderThread:85950 [sender.py:send_request():316] send_request: stop_status
2022-09-27 14:48:18,117 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: poll_exit
2022-09-27 14:48:18,700 INFO    Thread-12 :85950 [dir_watcher.py:_on_file_created():275] file/dir created: /home/xush/Project/AICompetition/feedback-prize/code/wandb/run-20220927_144816-3979rmin/files/conda-environment.yaml
2022-09-27 14:48:18,701 INFO    Thread-12 :85950 [dir_watcher.py:_on_file_created():275] file/dir created: /home/xush/Project/AICompetition/feedback-prize/code/wandb/run-20220927_144816-3979rmin/files/wandb-summary.json
2022-09-27 14:48:18,701 INFO    Thread-12 :85950 [dir_watcher.py:_on_file_created():275] file/dir created: /home/xush/Project/AICompetition/feedback-prize/code/wandb/run-20220927_144816-3979rmin/files/requirements.txt
2022-09-27 14:48:18,701 INFO    Thread-12 :85950 [dir_watcher.py:_on_file_created():275] file/dir created: /home/xush/Project/AICompetition/feedback-prize/code/wandb/run-20220927_144816-3979rmin/files/wandb-metadata.json
2022-09-27 14:48:19,056 INFO    Thread-16 :85950 [upload_job.py:push():138] Uploaded file /tmp/tmpnwtojt28wandb/2n2yiwe1-wandb-metadata.json
2022-09-27 14:48:23,501 DEBUG   SenderThread:85950 [sender.py:send():302] send: telemetry
2022-09-27 14:48:23,502 DEBUG   SenderThread:85950 [sender.py:send():302] send: exit
2022-09-27 14:48:23,502 INFO    SenderThread:85950 [sender.py:send_exit():438] handling exit code: 0
2022-09-27 14:48:23,503 INFO    SenderThread:85950 [sender.py:send_exit():440] handling runtime: 0
2022-09-27 14:48:23,507 INFO    SenderThread:85950 [sender.py:_save_file():1156] saving file wandb-summary.json with policy end
2022-09-27 14:48:23,507 INFO    SenderThread:85950 [sender.py:send_exit():446] send defer
2022-09-27 14:48:23,508 DEBUG   SenderThread:85950 [sender.py:send_request():316] send_request: poll_exit
2022-09-27 14:48:23,509 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: defer
2022-09-27 14:48:23,509 INFO    HandlerThread:85950 [handler.py:handle_request_defer():164] handle defer: 0
2022-09-27 14:48:23,510 DEBUG   SenderThread:85950 [sender.py:send_request():316] send_request: defer
2022-09-27 14:48:23,510 INFO    SenderThread:85950 [sender.py:send_request_defer():455] handle sender defer: 0
2022-09-27 14:48:23,510 INFO    SenderThread:85950 [sender.py:transition_state():459] send defer: 1
2022-09-27 14:48:23,511 INFO    MainThread:85950 [internal.py:handle_exit():80] Internal process exited
2022-09-27 14:48:23,512 DEBUG   HandlerThread:85950 [handler.py:handle_request():141] handle_request: defer
2022-09-27 14:48:23,512 INFO    HandlerThread:85950 [handler.py:handle_request_defer():164] handle defer: 1
2022-09-27 14:48:23,513 WARNING MsgRouterThr:85950 [router.py:message_loop():73] EOFError seen in message_loop
